src/components/shared/Drawer/index.js

```diff
@@ -1,96 +1,96 @@
-import React, { useReducer } from "react";
-import { Drawer, Fab, Hidden, Box } from "@material-ui/core";
+import React from "react";
+import { useRef, useReducer } from "react";
+import { motion, useCycle } from "framer-motion";
+import { useDimensions } from "./use-dimensions";
+import { MenuToggle } from "./MenuToggle";
+import { Navigation } from "./Navigation";
+import { Fab } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
 import MenuIcon from "@material-ui/icons/Menu";
 import CloseIcon from "@material-ui/icons/Close";
-import ListItems from "./ListItems";
-import DrawerFooter from "./DrawerFooter";
-import Logo from "../../../components/shared/Logo";
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    position: "absolute",
+    top: 0,
+    left: 0,
+    bottom: 0,
+    width: "300px",
+    background: "#fff",
+  },
+  nav: {
+    position: "fixed",
+    top: 0,
+    left: 0,
+    bottom: 0,
+    width: "300px",
+    zIndex: 1002,
+    pointerEvents: "none",
+  },
+  button: {
+    cursor: "pointer",
+    position: "absolute",
+    [theme.breakpoints.down("sm")]: {
```

Comment:
Dunno if this is copypasta stuff?

Since this is the same at both breakpoints, do we need this extracted into confusion/overhead of the makeStyles function? We can just take these and put them directly in the classname area?

Reply:
It isn't copypasta here - the `motion.nav` and `motion.div` components are not Material, and the example I was working off of used plain old CSS to style those, so I adapted it and copied it in that way. The button component you specifically referenced there is a Material UI `Fab` component, but it wasn't when I originally created it, so that's why I was still working in JSS.

But as this isn't done, the plan was/is to wrap them in Material UI boxes or some other component to control the styling consistent with what we've done throughout the app.